requirements.txt

```diff
@@ -0,0 +1 @@
+uproot
```

Comment:
Should we add here more requirements such as `numpy`, `pandas`, `pytables`, `notebook`, `matplotlib`, etc...?

Reply:
yes, so far I didn't need it but I will as times goes :)